Basic/APIDiorama/README.md

```diff
@@ -0,0 +1,15 @@
+# API Diorama
```

Comment:
A minor and a personal preference but I'd suggest calling the project "Examples", "Demos" or "Showcase" instead of "API Diorama" since they sound more common and obvious compared to "Diorama" and I believe we should omit "API" from the project name entirely anyway.

Epic/Unreal calls [a similar project as "Content Examples"](https://unrealengine.com/marketplace/en-US/product/content-examples) which I think is a good choice.

So, if I understood the main purpose of this project correctly, I'd like to propose "**NetcodeSamples**", similar to [other DOTS samples projects](https://github.cds.internal.unity3d.com/unity/dots/tree/master/Projects).

Reply:
A minor and a personal preference but I'd suggest calling the project "Examples", "Demos" or "Showcase" instead of "Diorama" since they sound more common and obvious compared to "Diorama".

So, I'd like to propose "API Showcase" as my personal pick but let's have this discussion.